feature_engine/selection/drop_psi_features.py

```diff
@@ -251,6 +240,7 @@ def __init__(
         missing_values: str = "raise",
         variables: Variables = None,
         confirm_variables: bool = False,
+        p_value: float = 0.001,
```

Comment:
I would suggest to add this parameter on a different order, for example after threshold, to show they are more closely related. Moreover usually feature engine selector ends with the parameters `variables` and `confirm_variables`.

Reply:
Not sure about this comment, as adding it at the end works better with backwards compatibility. I think the maintainers should have the final say on it.

Reply:
Interesting discussion. But if we put it after threshold, users that are already familiar with the order of the params will be frustrated. Let's leave it for the time being.